index.js

```diff
@@ -739,9 +704,16 @@ RedisClient.prototype.send_command = function (command, args, callback) {
 
     // if the value is undefined or null and command is set or setx, need not to send message to redis
     if (command === 'set' || command === 'setex') {
-        if(args[args.length - 1] === undefined || args[args.length - 1] === null) {
+        if (args.length === 0) {
```

Comment:
I'm uncertain if we want to support this behavior but this is how the tests behave right now. I'd say it's very unlikely that any user is going to call `set` / `setex` without any arguments. They might be undefined, but that would result in an error...